src/main/java/frc/robot/RobotContainer.java

```diff
@@ -37,23 +39,24 @@ public class RobotContainer {
     private final Controller controller = new Controller(OperatorConstants.kDriverControllerPort);
 
     private final DriveSubsystem driveSubsystem = new DriveSubsystem();
-    private final WristSubsystem wristSubsystem = new WristSubsystem();
-    // private final ShooterSubsystem shooterSubsystem = new ShooterSubsystem();
-
-    // private final DriveCommand driveCommand = new DriveCommand(driveSubsystem, controller);
+    private final DriveCommand driveCommand = new DriveCommand(driveSubsystem, controller);
+  
+    private final CoralIntakeSubsystem coralSubsystem = new CoralIntakeSubsystem();
```

Comment:
Move this to the top with the driveSubsystem. I want the structure of the file to be:

Subsystem declarations
Command declarations